lib/module.js

```diff
@@ -15,6 +18,7 @@ const {
   enableCompileCache,
   flushCompileCache,
   getCompileCacheDir,
+  getUserConditions,
```

Comment:
Is this helper actually helpful? I don't really see the point to have the wrapping `SafeSet` tbh